tests/test_engine.py

```diff
@@ -1,23 +1,23 @@
 import os
 from nose.tools import raises, eq_
-from moban.engine import EngineFactory, Engine, Context
+from moban.engine import ENGINES, Engine, Context
 import moban.exceptions as exceptions
 from moban.extensions import jinja_global
 
 
 def test_default_template_type():
-    engine_class = EngineFactory.get_engine("jinja2")
+    engine_class = ENGINES.get_engine("jj2")
```

Comment:
refer to other comment about keeping the engine name as "jinja2"

Reply:
restored